docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -127,6 +128,36 @@ To create an export configuration, do the following:
 1. On the **Integrations** page, click **Configure** for the Prometheus provider or, if a configuration is already available, **Add Configuration**.
 1. Enter a name for the configuration.
 1. Enter the endpoint URL of the Prometheus instance.
+1. Click **Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="VictoriaMetrics" lang="victoria" %}}
+
+VictoriaMetrics integration is {{<badge/tp>}} and only available for clusters deployed on AWS.
+
+The VictoriaMetrics integration requires the following:
+
+- VictoriaMetrics instance
+  - deployed in a VPC on AWS
+  - publically-accessible endpoint URL that resolves to the private IP of the VictoriaMetrics instance; the DNS for the endpoint must be in a public hosted zone in AWS.
+  - VPC hosting the VictoriaMetrics instance has the following Inbound Security Group rules:
```

Comment:
Can we add another point here(or somewhere else) specifying that url entered must be of the form specified by VictoriaMetrics docs here and here https://docs.victoriametrics.com/single-server-victoriametrics/#sending-data-via-opentelemetry
https://docs.victoriametrics.com/guides/getting-started-with-opentelemetry/